### PR TITLE
Allows light replacers to fit in tool-belts

### LINKED ISSUE #14768: Light replacers dont fit in toolbelts as advertised.
#### Description of issue
In a toolbelt's item text upon inspection it states that a light replacer fit inside of it but upon trying to place one inside you find out they don't fit.

#### Difference between expected and actual behavior
Light replacer gets a too big messege instead of being placed inside.

#### Steps to reproduce 
Try to put a light replacer in a toolbelt

#### Server revision
Revision: 7b496ed8fdeb170c69a5033cb6012634fd5f4ad4

#### Issue bingo

Please check whatever applies. More checkboxes checked increase your chances of not being yelled at by every contributor.

- [x] Issue could be reproduced at least once
- [ ] Issue could be reproduced by different players
- [x] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this](https://github.com/d3athrow/vgstation13/issues)

### PR DESCRIPTION
**Pull request by ghost**
~~Removes light replacers from the examine list, seeing as they apparently have a w_class too high to fit inside, but leaves them in the can_only_hold list because Kurfusten said so and I trust him more than myself.~~

~~Tosses in a fits_max_w_class var for the tool-belt because max_combined_w_class doesn't actually do anything tangible, apparently. Doesn't break anything, either, surprisingly!~~

I hijacked this PR and did it the correct way since OP dropped off the face of the earth.

Closes #14768

:cl:
* tweak: Toolbelts now fit light replacers.

### CODE REVIEW

**Comment by ghost** [comment]
just increase the total_max_w_class for the belt fam

**Comment by ghost** [comment]
I mean I _guess_ but like I said I'm just doing what Kurfusten said because he's cool.

I guess that wouldn't hurt.

**Review by PJB3005 — APPROVED** [review]

**Comment by Kurfursten** [comment]
This change as it is now would allow normal toolbelts to hold foam extinguishers, plus enlarged tank welders. So it's a categorical buff.

File: code/game/objects/items/weapons/storage/belt.dm [hunk 1]
```diff
@@ -32,6 +32,7 @@
 	w_class = W_CLASS_LARGE
 	storage_slots = 14
 	max_combined_w_class = 200 //This actually doesn't matter as long as it is arbitrarily high, bar will be set by storage slots
+	fits_max_w_class = 3
```
Comment (ComicIronic) [inline code/game/objects/items/weapons/storage/belt.dm]: You should use w_class defines for this if you're going to make this kind of change.

**Comment by ghost** [comment]
I will reserve my support, because I would personally like to see said tool-belt buff.

**Comment by SonixApache** [comment]
tg/baycode inventory sizes when

**Comment by ComicIronic** [comment]
We need that bot on this repo.
```
W
Y
C
I
```

**Comment by JustSumBody** [comment]
powercreep

File: code/game/objects/items/weapons/storage/belt.dm (hunk 1 above)
Comment (ghost) [reply code/game/objects/items/weapons/storage/belt.dm]: I would if I knew how to.
Unless you mean just changing the light replacer's w_class, except that doesn't seem like a good idea.

Comment (ComicIronic) [reply code/game/objects/items/weapons/storage/belt.dm]: There are defines for all the w_class values. You can see that the belt's w_class is `W_CLASS_LARGE`, for example. 3 is `W_CLASS_MEDIUM`, I believe.

**Comment by FudgePucker** [comment]
@Kurfursten can't you just specify the pocket extinguishers instead of the parent extinguisher?

@JustSumBody "I have no real valid argument against this PR but still disagree with it." ITS THAT WORD, THAT MAGIC WORD

**Comment by Kurfursten** [comment]
You could, but that's not what's being done here. By contrast there's no easy solution for the welder problem.

I think it's unfair to last out at SumBody like that, though: he's right. We've always had special rewards for using the CE belt, to encourage people to play that role and unite the engineering team. By making light replacers, foam extinguishers, and large-tank welders more widely available in toolbelts we are decreasing the substantialness of the reward for using the CE belt.

Since we have a Janibelt now specifically for holding stuff like the light replacer, it feels more than ever that engineers should leave lights to the janitor.

**Comment by FudgePucker** [comment]
I guess it would be too much to change all instances of welders with a new welder copy to move away from having the stock welder being the parent?

That's a fair argument.
I just don't like the power creep argument.
I don't think a single PR will result in a downward spiral of other game breaking PRs using the first PR as an excuse.
Its the slippery slope argument, only worse.

**Comment by Exxion** [comment]
Yeah I know powercreep is a buzzword
But this is like... the definition of powercreep

**Comment by Shadowmech88** [comment]
Isn't the light replacer pretty much just a janitor thing?
Janitors have janibelts now, which can already hold light replacers.

Why buff normal toolbelts as well?

**Comment by ghost** [comment]
I'd love to free up more space in my toolbox so I can carry more minerals, boxes, wires and cells.

Nah, eng has to do it a lot too so I end up stealing it or making one once minerals drop.

**Comment by SonixApache** [comment]
CE's belt still holds the RPD and RCD, unlike normal belts. That's reason enough to use it.

File: code/game/objects/items/weapons/storage/belt.dm (hunk 1 above)
Comment (Intigracy) [inline code/game/objects/items/weapons/storage/belt.dm]:
```
fits_ignoring_w_class = list(
		"/obj/item/device/lightreplacer"
		)
```

remove the change you made

**Review by Intigracy — CHANGES_REQUESTED** [review]
If you want to add the light replacer, add just the light replacer.

**Comment by Intigracy** [comment]
Make the change if you want this merged

**Comment by SonixApache** [comment]
@Emilc4h status?

**Comment by Shadowmech88** [comment]
Couldn't a collab just make the necessary change to the existing PR using the web interface and then merge it? It's only a single line.
Or did he not allow maintainers to edit the PR?

**Comment by Intigracy** [comment]
Muh self merge

**Review by Intigracy — APPROVED** [review]

**Comment by SonixApache** [comment]
@skullyton

**Pull request merged**